### PR TITLE
Create dash_google_auth package

### PR DESCRIPTION
**Pull request by joshbode**
Initial packaging `dash_google_auth` so that code can be used more easily in other projects:

e.g. Install package in editable mode:
```
$ git clone https://github.com/lucaschapin/dash-google-auth.git
...
$ pip install -e dash-google-auth
...
$ python
>>> from dash_google_auth import GoogleOAuth
...
```

Includes set up for TravisCI to push releases to PyPI

### CODE REVIEW

**Comment by joshbode** [comment]
Added `.travis.yml` with tests and deployment to PyPI on tag (currently using my account `joshbode` as a placeholder)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- '3.7'
+install:
+- pip install -r requirements.txt
+script:
+- pytest
+deploy:
+  provider: pypi
+  user: joshbode
+  password:
+    secure: XLoBRIUZti8jb+G9lD1z4IvIMVRKyR41PbnPPPQ2W8yxiSxawBEC4igUby3vfWQAqY8oiVV0aYhxld+EHucdvfY1lnk0UmWOS+eogCXfY0cDkaENH1XIdMLGMLWcibG4USoBDGSUuu0g3IbREkr6xDGeV6123Sd9AS+T1I9/Dmq9fzPMc4a817Cyu+w6+gQd//pjkZ3Ikdlif6cjMIYgAvuxIkGBeykxwqAl5iDq+a6gdJnrLGDiY/MwXKiuamBecaf/mArr/YCdPw6oP5ZiYZb6JILxhONyfqlt4AxgH26EPl5Ctz+HxMjWoSl6CGc1TF/zJZzXQp4qiC7WFwMY4ZCOQGrkuWd7f/JbWZM27YRWxG20VCpnKhL2KAICpi7EpFMfphiMZ08tSiPVGRQchmXPsbcO5kn7xMHxvHEFfLT9i7bsWFJroDtBmgqOPpzR4iE2le3qJi7AViX4O8irBKqAYCCoZhIJzRdTcgeFlGiIW6JI4EqAXV7zdTAhIvTHqpEUT1uVxUInarL/zkj+INzjY6fqu0insz6FwQ4YWk8frsWlhLKSaVTJQI0jREDW6K8VlFVOmkgBQEk1lZk5JXHun6A7Mh7p3L2VJx7LMHFdllKuYOdfO/cP7bzfMSgNmfgHhk8ONuYBNGQ2m1i1kd5wun2/BgsI5zUCvYy7Ok8=
```
Comment (brylie) [inline .travis.yml:15]: Should this be in Git?

Comment (joshbode) [reply .travis.yml:15]: As in am I giving everyone my password?
If that's what you mean, then it's OK - it is encrypted for TravisCI (see https://docs.travis-ci.com/user/encryption-keys/)

Comment (joshbode) [reply .travis.yml:15]: But this should probably use an account for the package (or "someone in charge")

**Comment by lchapo** [comment]
@joshbode I'm good with this. One request - could you update the readme as well to reflect the changes?

I added you as a collaborator so you can merge yourself when you're ready.

**Comment by joshbode** [comment]
Thanks @lucaschapin! I've updated the README - let me know if you like the changes and I'll merge :)

**Comment by joshbode** [comment]
you might need to enable the repo in https://travis-ci.org also

**Comment by lchapo** [comment]
Looks great, thanks! I authorized Travis CI as well.

**Comment by joshbode** [comment]
Aw yeah:
https://pypi.org/project/dash-google-auth/

Still working on fixing the build - the secure password is failing, but I uploaded the package manually :)

**Comment by lchapo** [comment]
Nice job!!

**Pull request merged**